frontend/src/assets/css/loader.scss

```diff
@@ -0,0 +1,82 @@
+$loader-bg-color: #ffffff80;
```

Comment:
We use scss modules for components styles, so rename this file to styles.module.scss and move inside loader component folder. Then you will be able to use styles like this - `className={styles["style-name"]}`